docs/readme.md

```diff
@@ -71,6 +71,12 @@ Support
 
 Release Notes
 -------------
+### Upcoming
```

Comment:
still upcoming?

Reply:
Yeah, not updating the version number of the Unity SDK yet, so upcoming (though hopefully not too much longer)

Reply:
Looking forward to it! We really, really need the updated versions. Thanks!

Reply:
Sorry for the delays, we are still fixing a problem with the Android dependencies, which we are still testing and iterating against different Unity editor versions, but again, hopefully will be available soon.